core/src/main/java/com/gentics/mesh/core/data/job/impl/NodeMigrationJobImpl.java

```diff
@@ -60,10 +61,10 @@ private SchemaMigrationMeshEventModel createEvent(MeshEvent event, JobStatus sta
 		return model;
 	}
 
-	private NodeMigrationActionContextImpl prepareContext() {
```

Comment:
Ensure that MicronodeMigration and BranchMigration all work the same way. Either return Single or the context.